packages/moonstone/styles/mixins.less

```diff
@@ -392,6 +392,7 @@
 	.font-kerning;
 }
 
+// NOTE: Consider deprecation in 3.0.0
```

Comment:
I intended this message as a reminder for removal from 3.0.0. (so it would be deprecated in 2.x). However, we don't know the extent of the usage in apps or other scenarios, so more research should be done before we officially drop support for this mixin.